jib-core/src/main/java/com/google/cloud/tools/jib/frontend/BuildStepsRunner.java

```diff
@@ -190,23 +187,16 @@ public void build(HelpfulSuggestions helpfulSuggestions) throws BuildStepsExecut
       // Logs the different source files used.
       buildLogger.info("Containerizing application with the following files:");
 
-      buildLogger.info("\tClasses:");
-      buildSteps
-          .getSourceFilesConfiguration()
-          .getClassesFiles()
-          .forEach(classesFile -> buildLogger.info("\t\t" + classesFile));
-
-      buildLogger.info("\tResources:");
-      buildSteps
-          .getSourceFilesConfiguration()
-          .getResourcesFiles()
-          .forEach(resourceFile -> buildLogger.info("\t\t" + resourceFile));
-
-      buildLogger.info("\tDependencies:");
-      buildSteps
-          .getSourceFilesConfiguration()
-          .getDependenciesFiles()
-          .forEach(dependencyFile -> buildLogger.info("\t\t" + dependencyFile));
+      for (LayerConfiguration layerConfiguration :
+          buildSteps.getBuildConfiguration().getLayerConfigurations()) {
+        buildLogger.info("\t" + capitalizeFirstLetter(layerConfiguration.getLabel()) + ":");
```

Comment:
I mostly had this to keep the log message consistent with what it was before. We should definitely consider changing this logic to be something more configurable, but this log message will be part of the "`plugins-common`" rather than `jib-core`. There should probably be an enum from a layer type (DEPENDENCIES, CLASSES, etc.) to the intended label string and log string.

Reply:
Alright, just keep in mind you'd normally consider what locale to use if the text can be something else than English, e.g., https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#toUpperCase(java.util.Locale)